### PR TITLE
Source Jira: fixing schemas accordinately to response.

### LINKED ISSUE #4341: Jira Connector declares "created" cursor field but does not include it in its catalog
#### users (Mode:  Full refresh - Overwrite) 
But while synching connector we are getting error "Caused by: io.airbyte.workers.WorkerException: Normalization Failed".
Attached complete logs for reference. 

#### Can someone help me to resolve this issue ?
[logs-232-2.txt](https://github.com/airbytehq/airbyte/files/6714758/logs-232-2.txt)

### PR DESCRIPTION
**Pull request by vovavovavovavova**
## What
Closes  #4341 and #4950   

## How
By fixing all the schemas that are currently using in the Jira connector. Current version can produce
`KeyError: "'json_schema'.'properties' are not defined for stream project_components"` Errors. Dev version suggested in this pr does not have that, at least in my case.

## Recommended reading order
1. `x.java`
2. `y.python`

### CODE REVIEW

File: airbyte-integrations/connectors/source-jira/integration_tests/inc_configured_catalog.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "issues",
+        "json_schema": {},
+        "supported_sync_modes",: ["incremental"],
```
Comment (keu) [inline airbyte-integrations/connectors/source-jira/integration_tests/inc_configured_catalog.json:7]: ?

File: airbyte-integrations/connectors/source-jira/source_jira/schemas/application_roles.json [hunk 2]
```diff
@@ -1,57 +1,48 @@
...
+    },
+    "defined": {
+      "type": "boolean"
     },
     "groups": {
-      "uniqueItems": true,
-      "type": "array",
-      "description": "The groups associated with the application role.",
-      "items": { "type": "string" }
+      "items": {
```
Comment (keu) [inline airbyte-integrations/connectors/source-jira/source_jira/schemas/application_roles.json:14]: format

**Review by keu — COMMENTED** [review]
comments

File: airbyte-integrations/connectors/source-jira/source_jira/schemas/application_roles.json (hunk 2 above)
Comment (vovavovavovavova) [reply airbyte-integrations/connectors/source-jira/source_jira/schemas/application_roles.json:14]: applied but somewhy it does not available in commit diff (available in global diff)

**Review by keu — APPROVED** [review]

**Pull request merged**